docs/getting-started/supported-languages-and-tools.md

```diff
@@ -72,6 +77,9 @@ The table below lists all languages that Codacy supports and the corresponding t
       <td><a href="https://github.com/bridgecrewio/checkov/">Checkov</a></td>
       <td>-</td>
       <td><a href="https://github.com/bridgecrewio/checkov/">Checkov</a>,
+          <!--
+          <a href="https://semgrep.dev/">Semgrep</a> <a href="#yaml-only"><sup>2</sup></a>,
```

Comment:
to uncomment this we need to first add Language.YAML as a supported language to Semgrep in codacy/codacy-tools